### PR TITLE
Add import-order hook and fix import order (close #102)

### LINKED ISSUE #102: Constrain keep import order with hook and CI
Use [import-order](https://github.com/spoqa/import-order), we can enforce import order.

just install and run `import-order cliche`.

I think we use it in git commit hook and CI

### PR DESCRIPTION
**Pull request by item4**
Title is all.

I think first CI running will be failed. because of `pathlib` or `enum`. I check it how it warn it and fix it.

### CODE REVIEW

File: hooks/pre-commit [hunk 1]
```diff
@@ -1,4 +1,14 @@
 #!/bin/bash
 set -e
+if [[ $TRAVIS_PYTHON_VERSION ]]; then
+    VERSION=$TRAVIS_PYTHON_VERSION
+else
+    VERSION=$(python3 -c 'import sys; v = sys.version_info; print("{}.{}".format(v[0], v[1]));')
+fi
 python3 -m flake8 -j auto
+if [ $VERSION = '3.4' ]; then
```
Comment (miaekim) [inline hooks/pre-commit]: why don't you check 'import order' in python 3.3?

Comment (item4) [inline hooks/pre-commit]: @miaekim we use py34 mainly. but py33 is not include some standard library such as `enum`, `pathlib`. so we install polyfill from pypi. polyfill is not std lib, so import-order can not decide correct order.

**Comment by miaekim** [comment]
It looks fair. @clicheio/owners, any comments?

**Comment by miaekim** [comment]
LGTM!
[LGTM](http://www.lgtm.in/i/YQlwm7r1q)

[:+1:](http://www.lgtm.in/u/YQlwm7r1q) [:-1:](http://www.lgtm.in/r/YQlwm7r1q)

**Pull request merged**